### ISSUE TITLE
Align background with expanded text

### ISSUE THREAD

**Issue by dmfs** [opacapp/multiline-collapsingtoolbar#35]
How can I add a background shape/color and align it with the expanded title like so?
![screenshot_1491245756](https://cloud.githubusercontent.com/assets/1614987/24626292/16c424c6-18b1-11e7-8d12-8e1ba238f57d.png)
The goal is that the background expands/shrinks with the title.

**Comment by raphaelm** [comment]
Isn't this a standard feature of collapsing toolbar? I'm not an expert here. How is this related to our library?

**Comment by johan12345** [comment]
@raphaelm I think he is talking about the gray overlay behind the text. That's not a standard feature of the collapsing toolbar.

It would probably be possible to create such a background with a fixed height (using a semi-transparent view inside the CollapsingToolbarLayout). But if you want the height to adapt to the text height (also during the animation between collapsed and expanded states), the easiest way is probably to directly look at the drawing code.
At [CollapsingTextHelper.java:553](https://github.com/opacapp/multiline-collapsingtoolbar/blob/master/multiline-collapsingtoolbar/src/main/java/net/opacapp/multilinecollapsingtoolbar/CollapsingTextHelper.java#L553), there's some code that draws a rectangle behind the text used for debugging. Probably that could be adapted to your use case.

**Comment by dmfs** [comment]
@johan12345 yes, that's what I meant. Since the text doesn't have a fixed height, it's not possible to align it with any existing view.

I'll check out the code you referred to. Maybe I can build something generic to align any existing view with the text
